pkgs/development/libraries/sundials/3.x.lapack.nix

```diff
@@ -0,0 +1,33 @@
+{ cmake, fetchurl, python, stdenv, blas, liblapack, gfortran }:
+
+stdenv.mkDerivation rec {
+
+  pname = "sundials";
+  version = "3.2.1";
+
+  src = fetchurl {
+    url = "https://computation.llnl.gov/projects/${pname}/download/${pname}-${version}.tar.gz";
+    sha256 = "0238r1qnwqz13wcjzfsbcfi8rfnlxcjjmxq2vpf2qf5jgablvna7";
+  };
+
+
+  cmakeFlags = [
+    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples"
+    "-DBLAS_ENABLE=ON" "-DSUNDIALS_INDEX_TYPE=int32_t"
+    "-DBLAS_LIBRARIES=[${blas}/lib/libblas.a,${gfortran.cc.lib}/lib/libgfortran.a]"
+    "-DLAPACK_ENABLE=ON"
+    "-DLAPACK_LIBRARIES=[${liblapack}/lib/liblapack.a,,${gfortran.cc.lib}/lib/libgfortran.a]"
```

Comment:
we should see if we can use `.so`/`.dylib` here (and make it more generic via `stdenv.hostPlatform.extensions.sharedLibrary`, instead of `.a`.